chkconfig.spec

```diff
@@ -95,11 +95,12 @@ mkdir -p $RPM_BUILD_ROOT/etc/chkconfig.d
 %files -n alternatives
 %license COPYING
 %dir /etc/alternatives
+%ghost /etc/alternatives.admindir
```

Comment:
This still needs to be `%dir`. That is:
```
%ghost %dir /etc/alternatives.admindir
```
and the same for the next line.

Reply:
good catch, but I found out it is not enough, it also needs attr for -qV to be happy